react-js/styling-react.md

```diff
@@ -0,0 +1,109 @@
+# Styling in React
+
+### Projected Time : 1 hr
+- Lesson: 40 mins.
+- Guided Practice: 20 mins.
+
+### Prerequisites
+- [CSS Lesson](/css/css)
+- [Bootstrap Lesson](/bootstrap/bootstrap)
+- [React Lesson](react)
+
+
+### Motivation
+
+Styling has been one of the building blocks of web development. Cascading Style Sheets is a style sheet language used for describing the presentation of a document written in a markup language like HTML. React only provides a virtual DOM, whereas the presentation part still lies with CSS.
+
+### Objectives
+
+- Styling in JS Framework.
+- Inline styling and using external Stylesheet.
+- Bootstrap in React.	 
+
+### Specific Things To Learn
+
+- Defining styles as objects.
+- Importing stylesheets into react application.
+
+### Materials
+
+- [Video: Styling of React App in 3 different ways](https://www.youtube.com/watch?v=j5P9FHiBVNo) - Using external CSS, internal and CSS modules.
+- [Video: Bootstrap in React](https://www.youtube.com/watch?v=wdbzLi0tWq0) - Bootstrap in React can be used using CDN or by installing the package. 
+
+### Lesson
+
+- [Slides: CSS in React ](https://drive.google.com/open?id=15oauTP4afibqMvNKW5XgVC1qraYUKxY-)
+
+### Guided Practice
+> #### Creating a simple div with InLine and External CSS and converting it into Bootstrap.
+
+1. Inline And External CSS
+  - App.js
+    - Creating a simple div using and add custom styling to it.
+    - ```
+      <div className="my-div">
+		<div style={{backgroundColor:"red"}}>Inner Div 1</div>
+		<div style={{backgroundColor:"blue"}}>Inner Div 2</div>
+		<div style={{backgroundColor:"green"}}>Inner Div 3</div>
+	  </div>
+      ```
+  - CSS 
+    - Make the *div* inside *my-div* class as flex box and add a little bit of padding.
+    - ```
+       .my-div
+	   {
+	     margin: 10px 0;
+	     display: flex;
+	     width: 100vw;
+	     background-color: darkorange;
+       }
+      .my-div div
+	  {
+	   flex-basis: 25%;
+	   max-width: 25%;
+	   padding: 10px;
+      }
+      ```
+2. Using Bootstrap
+  - App.js
```

Comment:
Please tell them what they are doing next and why. “Using bootstrap” isn’t enough. How will they know that it worked? Should they change the same file or make a new one?